### PR TITLE
wssecurity feature updates

### PR DESCRIPTION
**Pull request by arunavemulapalli**
update wssecurity feature and runtime to work with new cxf/wss4j versions

### CODE REVIEW

**Comment by arunavemulapalli** [comment]
#run-libby-bot

**Comment by arunavemulapalli** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.wssecurity_fat.cxf,com.ibm.ws.wssecurity_fat.saml,com.ibm.ws.wssecurity_fat.saml.2

**Comment by arunavemulapalli** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.wssecurity_fat.cxf,com.ibm.ws.wssecurity_fat.saml,com.ibm.ws.wssecurity_fat.saml.2

**Comment by arunavemulapalli** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.wssecurity_fat.cxf,com.ibm.ws.wssecurity_fat.saml,com.ibm.ws.wssecurity_fat.saml.2

**Comment by arunavemulapalli** [comment]
#run-libby-bot

**Comment by andrea-pichler** [comment]
No L2 message review needed for translations

**Comment by arunavemulapalli** [comment]
#build

**Comment by arunavemulapalli** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.wssecurity_fat.cxf

**Comment by arunavemulapalli** [comment]
#run-libby-bot

**Review by neuwerk — COMMENTED** [review]
Other then the comments I made, I reviewed the code by comparing it to the original `com.ibm.ws.wssecurity` bundle and while it made sense in some ways, I'm no way familiar with it enough to say 100% that my review would be as good as someone reviewing the code that is familiar with WS-Security.

File: dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.wsSecurity1.1-jaxws2.3.feature [hunk 1]
```diff
@@ -0,0 +1,26 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.wsSecurity1.1-jaxws2.3
+IBM-App-ForceRestart: install, \
+ uninstall
+IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.jaxws-2.3))", \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.wsSecurity-1.1))"
+IBM-Install-Policy: when-satisfied
+-features= \
+  com.ibm.websphere.appserver.jaxws-2.3, \
+  com.ibm.websphere.appserver.internal.slf4j-1.7.7,\
```
[the remark this answers is not in the document]
Comment (arunavemulapalli) [reply dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.wsSecurity1.1-jaxws2.3.feature]: removed it

File: dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.wsSecurity1.1-jaxws2.3.feature [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  com.ibm.websphere.appserver.ssoCommon-1.0, \
+  com.ibm.websphere.appserver.wss4j-2.3, \
+  com.ibm.websphere.appserver.httpcommons-1.0
+-bundles= \
+  com.ibm.ws.org.joda.time.2.9.9, \
+  com.ibm.ws.prereq.wsdl4j.1.6.2, \
```
[the remark this answers is not in the document]
Comment (arunavemulapalli) [reply dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.wsSecurity1.1-jaxws2.3.feature]: removed it

**Comment by arunavemulapalli** [comment]
#run-libby-bot

**Comment by arunavemulapalli** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.wssecurity_fat.cxf

**Review by neuwerk — APPROVED** [review]

**Comment by arunavemulapalli** [comment]
This is required for #12449
This fixes #16458

**Pull request merged**